src/components/DataTableData/DataTableData.js

```diff
@@ -0,0 +1,63 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+
+const DataTableData = props => {
+  const { children, className, checked, value, id, expanded, ...other } = props;
```

Comment:
For prop names, should they be `isSelectable` and `isExpandable`? The rationale being that having those props adds in the behavior to be selected or expanded, but doesn't indicate any kind of state in this component (e.g. if it's selected or checked).

For selectable vs checked, the rationale being that there is a potential that selecting a role might not be tied to a checkbox UI control so would be an abstraction on top of that.

Reply:
I can switch it to `isSelectable` and `isExpandable`, that makes sense.